src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -1066,8 +1087,27 @@ int xcldev::device::validate(bool quick)
         return retVal;
 
     // Test verify kernel
-    retVal = runOneTest("verify kernel test",
-            std::bind(&xcldev::device::verifyKernelTest, this));
+    if (is_u50) {
```

Comment:
what is wrong with original code? does it not work on u50, so we need something special for u50?